Dynatrace e2e test is failing

The nightly test has been failing for several days, especially on the dynatrace e2e test.
It needs to be investigated why the test is failing.

Logging:

```
2025-08-20T01:11:23.5680224Z tests/scalers/dynatrace/dynatrace_test.go has failed after "three" attempts 
2025-08-20T01:11:23.5680720Z attempt number "one"
2025-08-20T01:11:23.5681211Z === RUN   TestDynatraceScaler
2025-08-20T01:11:23.5681778Z     dynatrace_test.go:***5: --- setting up ---
2025-08-20T01:11:23.5682380Z     helper.go:***60: deleting namespace dynatrace-test-ns
2025-08-20T01:11:23.5683029Z     helper.go:***4: waiting for namespace dynatrace-test-ns deletion
2025-08-20T01:11:23.5683630Z     helper.go:***47: Creating namespace - dynatrace-test-ns
2025-08-20T01:11:23.5684640Z     helper.go:575: Applying template: secretTemplate
2025-08-20T01:11:23.5685293Z     helper.go:575: Applying template: triggerAuthenticationTemplate
2025-08-20T01:11:23.5685890Z     helper.go:575: Applying template: deploymentTemplate
2025-08-20T01:11:23.5686487Z     helper.go:575: Applying template: scaledObjectTemplate
2025-08-20T01:11:23.5687497Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - 0
2025-08-20T01:11:23.5688086Z     dynatrace_test.go:***58: --- testing activation ---
2025-08-20T01:11:23.5688875Z     helper.go:5***8: Waiting for some time to ensure deployment replica count doesn't change from 0
2025-08-20T01:11:23.5689545Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5690220Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5690882Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5691907Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5692637Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5693310Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5693978Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5694645Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5695317Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5695989Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5696658Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5697324Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5697990Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5698674Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5699350Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5700014Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5700683Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5701357Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5702027Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5702691Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5703361Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5704025Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5704717Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5705417Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5706090Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5706759Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5707444Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5708109Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5708779Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5709459Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5710129Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5710799Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5711471Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5712590Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5713258Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5713933Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5714611Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5715281Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5715963Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5716631Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5717302Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5717979Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5718647Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5719588Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5720552Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5721258Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5721932Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5722599Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5723274Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5723947Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5724614Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5725284Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5725954Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5726645Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5727321Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5727994Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5728662Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5729332Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5730000Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5730668Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5731338Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5732004Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5732675Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5733368Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5734043Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5734715Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5735395Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5736072Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5736744Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5737414Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5738085Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5738755Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5739951Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5740634Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5741306Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5741989Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5742663Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5743330Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5744005Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5744676Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5745347Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5746024Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5746707Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5747672Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5748414Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5749094Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5749766Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5750442Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5751114Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5751780Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5752450Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5753122Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5753804Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5754478Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5755150Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5755820Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5756492Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5757170Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5757842Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5758511Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5759186Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5759856Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5760661Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5761378Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5762046Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5762712Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5763388Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5764058Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5764730Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5765407Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5766078Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5766748Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5767920Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5768600Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5769273Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5769947Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5770617Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5771285Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5781954Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5782798Z     helper.go:5***5: Deployment - dynatrace-test-deployment, Current  - 0
2025-08-20T01:11:23.5783414Z     dynatrace_test.go:***66: --- testing scale out ---
2025-08-20T01:11:23.5784424Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5785826Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5786877Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5787872Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5788872Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5789861Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5790856Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5791861Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5792862Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5793848Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5794848Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5795841Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5796832Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5797834Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5798825Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5799814Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5801158Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5802184Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5803184Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5804205Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5805648Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5806669Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5807690Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5808694Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5809715Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5810739Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5812035Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5813101Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5814115Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5815116Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5816130Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5817130Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5818144Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5819163Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5820171Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5821180Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5822175Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5823179Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5824184Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5825203Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5826205Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5827219Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5828221Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5829221Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5830220Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5831676Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5832688Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5833690Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5834693Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5835706Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5836715Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5837960Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5839017Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5840016Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5841110Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5842130Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5843131Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5844162Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5845164Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5846188Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - ***
2025-08-20T01:11:23.5846705Z     dynatrace_test.go:***69: 
2025-08-20T01:11:23.5847455Z         	Error Trace:	/__w/keda/keda/tests/scalers/dynatrace/dynatrace_test.go:***69
2025-08-20T01:11:23.5848171Z         	            				/__w/keda/keda/tests/scalers/dynatrace/dynatrace_test.go:***5***
2025-08-20T01:11:23.5848696Z         	Error:      	Should be true
2025-08-20T01:11:23.5849241Z         	Test:       	TestDynatraceScaler
2025-08-20T01:11:23.5849887Z         	Messages:   	replica count should be *** after *** minutes
2025-08-20T01:11:23.5850501Z     dynatrace_test.go:***75: --- testing scale in ---
2025-08-20T01:11:23.5851532Z     helper.go:47***: Waiting for deployment replicas to hit target. Deployment - dynatrace-test-deployment, Current  - 0, Target - 0
2025-08-20T01:11:23.5852170Z     helper.go:644: Deleting template: scaledObjectTemplate
2025-08-20T01:11:23.5852792Z     helper.go:644: Deleting template: deploymentTemplate
2025-08-20T01:11:23.5853479Z     helper.go:644: Deleting template: triggerAuthenticationTemplate
2025-08-20T01:11:23.5854088Z     helper.go:644: Deleting template: secretTemplate
2025-08-20T01:11:23.5854708Z     helper.go:***60: deleting namespace dynatrace-test-ns
2025-08-20T01:11:23.5855380Z     helper.go:***4: waiting for namespace dynatrace-test-ns deletion
2025-08-20T01:11:23.5856071Z     helper.go:***4: waiting for namespace dynatrace-test-ns deletion
2025-08-20T01:11:23.5856741Z     helper.go:***4: waiting for namespace dynatrace-test-ns deletion
2025-08-20T01:11:23.5857302Z --- FAIL: TestDynatraceScaler (***8.4***s)
```

--- a/tests/scalers/dynatrace/dynatrace_test.go
+++ b/tests/scalers/dynatrace/dynatrace_test.go
@@ -131,7 +131,6 @@ spec:
 )
 
 func TestDynatraceScaler(t *testing.T) {
-	t.Skip("Disabled - Dynatrace tenant access is currently unavailable due to licensing; see https://github.com/kedacore/keda/issues/6995 for details")
 	// setup
 	t.Log("--- setting up ---")
 	require.NotEmpty(t, dynatraceToken, "DYNATRACE_METRICS_TOKEN env variable is required for dynatrace tests")

--- a/tests/scalers/dynatrace_dql/dynatrace_dql_test.go
+++ b/tests/scalers/dynatrace_dql/dynatrace_dql_test.go
@@ -132,7 +132,6 @@ spec:
 )
 
 func TestDynatraceDQLScaler(t *testing.T) {
-	t.Skip("Disabled - Dynatrace tenant access is currently unavailable due to licensing; see https://github.com/kedacore/keda/issues/6995 for details")
 	// setup
 	t.Log("--- setting up ---")
 	require.NotEmpty(t, dynatraceToken, "DYNATRACE_METRICS_TOKEN env variable is required for dynatrace tests")

